src/test/java/ch/uzh/ifi/hase/soprafs24/service/SessionServiceIntegrationTest.java

```diff
@@ -347,5 +347,30 @@ public void endSession_noExistingTeam() {
     // when (have something stored, but not that team)
     assertThrows(ResponseStatusException.class, () -> sessionService.endSession(99L));
   }
+
+  /*
+   * Test if the session is expired, that the 410 error is thrown
+   */
+  @Test
+  public void endSession_expiredSession_expectsError() {
+    // given
+    Team testTeam = new Team();
+    testTeam.setName("productiviTeam");
+    testTeam.setDescription("We are a productive team!");
+    testTeam.setTeamUUID("team-uuid");
+    teamRepository.saveAndFlush(testTeam);
+
+    // given expired session
+    Session testSession = new Session();
+    testSession.setTeam(testTeam);
+    testSession.setStartDateTime(LocalDateTime.now().minusDays(1));
```

Comment:
seems like a flaky test. maek minus days 1 and minus a few hours, just to be save. you never know with time tests

Reply:
you're correct